### DISCUSSION TITLE
Start colima on different disk

### DISCUSSION THREAD

**Discussion by santilococo** [abiosoft/colima#580] [Q&A]
Hello. I want to know if it is possible to start colima on an external disk. Something like `colima start --disk /Volumes/X`. That is because I want to download the docker's images on an external drive. Thanks!

**Comment by abiosoft** [comment]
That is not yet supported. The ability to specify custom directory for Colima is planned.

**Reply by VeganSpeak** [reply]
This would be fantastic for those of us with little internal disk space left but plenty external drive space.

**Reply by anarchy89** [reply]
Currently, is it possible to symlink the folder to an external drive?

**Reply by abiosoft** [reply]
@anarchy89 yeah you can symlink `~/.colima` to any external drive.

**Reply by rfay** [reply]
@abiosoft my bet is that people who want colima on an external drive actually want the large ~/.lima directory (where docker volumes and images are contained) on the external drive, and may not care about ~/.colima ?  (Do I remember discussion about moving the ~/.lima into ~/.colima so it's all contained in one place?)

**Reply by abiosoft** [reply]
@rfay oh yeah, you are very accurate. Thanks for pointing that out.

**Reply by anarchy89** [reply]
@abiosoft so can I just move the Colima and Lima folders to an external drive then symlink then back?

**Reply by abiosoft** [reply]
@anarchy89 that should work just fine. 

However, bear in mind that such symlinks would not be readable in the Docker containers.

**Reply by anarchy89** [reply]
@abiosoft ahh damn, are there plans for a fix to that as well?

**Reply by abiosoft** [reply]
You can modify the colima mounts to point to your external volume and that should work. But having a symlink in your $HOME that points to the external drive would not work in the VM.

**Reply by anarchy89** [reply]
so basically i need to put the full path to the external drive whenever i docker containers. Also, if I used brew to install colima on my macbook, the binary gets put in /opt/homebrew, are the .colima and .lima files that need to be moved still in my home directory ~/ ?

**Reply by abiosoft** [reply]
You need to modify colima mounts to include the external drives via the `--mount` flag or `mounts` config. The defaults are `$HOME` and `/tmp/colima`.

Nothing actually changes when you use Docker containers i.e. same docker commands.

However, if you were to symlink the external drive to somewhere in `$HOME`, a symlink is basically a pointer to another location, and the location would not exist in the colima VM, unless it is mounted as well.

So if you have your external drive in `/Volumes/myDrive`, you need to include that in the colima mounts. e.g.

```
colima start --mount $HOME:w --mount /Volumes/myDrive:w
```

**Reply by klodeckl** [reply]
What about mutagen when using mounts to external drives?

**Reply by mig8447** [reply]
I attempted to use the mount option this way and it didn't work

**Comment by dan-corneanu** [comment]
I have the same problem too.  
I have 3 colima instances that, combined, eat up something like 40G of drive. It would be nice to be able to specify the folder where to store the VM files for each colima instance.

**Comment by klausbegnis** [comment]
Hey there, was this solved already? any suggestions on how to specify the disk mount for Colima?

**Comment by abiosoft** [comment]
You can actually specify the `$COLIMA_HOME` env var to customize where Colima stores it's files.
